appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -94,9 +94,7 @@ export class CaseDetailsPage extends Page {
 	}
 
 	uploadOneDocument() {
```

Comment:
I'd recommend renaming this to better describe exactly what it's doing, such as `uploadSamplePdf()` for instance. As it gives an indication about the type of file being uploaded

This is useful as the test may be to try and upload an incorrect type of file (e.g .exe) to this information helps convey more information when reading the spec files